HerPublicWebsite/Views/Questionnaire/NotParticipating.cshtml

```diff
@@ -0,0 +1,111 @@
+﻿@using GovUkDesignSystem
+@using GovUkDesignSystem.GovUkDesignSystemComponents
+@using HerPublicWebsite.BusinessLogic.Models.Enums
+@using Microsoft.AspNetCore.Mvc.TagHelpers
+@using HerPublicWebsite.Controllers
+@using HerPublicWebsite.Models.Enums
+
+@model HerPublicWebsite.Models.Questionnaire.NotParticipatingViewModel
+
+@{
+    ViewBag.Title = "Your local authority is not taking part"; 
+}
+
+@section BeforeMain {
+    <nav>
+        @await Html.GovUkBackLink(new BackLinkViewModel
+        {
+            Text = "Back",
+            Href = Model.BackLink,
+        })
+    </nav>
+}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        
+        <h1 class="govuk-heading-l">Your Local Authority is not receiving applications via this service.</h1>
+        
+        <h2 class="govuk-heading-m">What to do next</h2>
+
+        @await Html.PartialAsync(Model.LocalAuthorityMessagePartialViewPath)
+
+        <p class="govuk-body">
+            If your local authority does not seem right, <a class="govuk-link" href="https://www.gov.uk/find-local-council">find your correct local authority</a> and contact them directly.
```

Comment:
⚠️ The agreed text for this page is "does not seem to be right" rather than just "does not seem right".